lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -417,6 +417,32 @@ def test_course_price_is_not_visble_in_sidebar(self):
         self.assertNotIn('<span class="important-dates-item-text">$10</span>', resp.content)
 
 
+@attr(shard=1)
+class AboutSidebarHTMLTestCase(SharedModuleStoreTestCase):
+    """
+    This test case will check the About page for the content in the HTML sidebar.
+    """
+    def setUp(self):
+        super(AboutSidebarHTMLTestCase, self).setUp()
+        self.course = CourseFactory.create()
+
+    def test_html_sidebar_empty(self):
+        url = reverse('about_course', args=[self.course.id.to_deprecated_string()])
+        resp = self.client.get(url)
+        self.assertEqual(resp.status_code, 200)
+        self.assertNotIn("About Sidebar HTML Heading", resp.content)
+
+    def test_html_sidebar_has_content(self):
+        ItemFactory.create(
+            category="about", parent_location=self.course.location,
+            data="About Sidebar HTML Heading", display_name="about_sidebar_html"
+        )
+        url = reverse('about_course', args=[self.course.id.to_deprecated_string()])
```

Comment:
nit: we should avoid using `to_deprecated_string` string in new code. I think we can use course id here
`args=[self.course.id]`